src/Umbraco.Core/Extensions/StringExtensions.cs

```diff
@@ -1558,6 +1559,14 @@ public static IEnumerable<string> EscapedSplit(this string value, char splitChar
         yield return sb.ToString();
     }
 
+    /// <summary>
+    ///     Checks whether a string is a valid email address.
+    /// </summary>
+    /// <param name="email">The string check</param>
+    /// <returns>Returns a bool indicating whether the string is an email address.</returns>
+    public static bool IsEmail(this string email) =>
+        new EmailAddressAttribute().IsValid(email);
```

Comment:
@Zeegaan should this do a bit more that using `EmailAddressAttribute`?
https://stackoverflow.com/questions/36947823/validating-an-email-string-in-net-using-emailaddressattribute-but-not-on-an-at

E.g. return `false` when value (email) is null, empty or whitespace?

Reply:
Yea you're probably right, looks like empty or whitespace is already covered, but I've added a null check 👍